_i18n/en.yml

```diff
@@ -0,0 +1,81 @@
+---
+global:
+  brazilian portuguese: Português
+  english: English
+header:
+  stories: Stories
+  explore: Explore
+  about: About
+support:
+  fund_us: Support the project
+home:
+  about: Artificial Intelligence for social control of public administration
+  about_rosie: Operação Serenata de Amor created a Robot - An artificial intelligence capable of analysing each request for reimbursement from congresspeople and identify it’s illegal probability.
+  last_histories: Last published histories
+  loading: Loading...
+  more_posts: More posts
+footer:
+  about: about
+  github: Serenata on github
+  contact: contact
+  partners: partners
+stories:
+  published: Published stories
+  loading: Loading...
+explore:
+  reports: Reports
+  about_reports: Rosie, our Robot, in numbers
+  tweet_about: Tweet about this number
+  chamber_reports: Reports done to the Chamber of Deputies
+  suspitions: Different congresspeople reported for suspicious reimbursements
+  questionated: thousand Brazilian Reals in suspicious expenses questioned
+  paragraph_one: Those values correspond to 371% of what we raised with our crowdfunding campaign (~U$ 25.000,00)
+  paragraph_two: Want to know how the report system works?
+  more: You can read more in this post
+about:
+  title: Serenata de amor operation.
+  who_contributed: People who contributed
+  paragraph_one: We are a project who make use of artificial intelligence to audit public accounts and fight corruption. The idea came from the data scientist
+  paragraph_two:  when he realized about the many possibilities to use technology to inspect congressperson’s spends. When he shared the idea with friends, the project growth in strength in no time and the team was formed with the goal to finish the project.
+  paragraph_three: Serenata de amor Operation was born, focused on inspection of reimbursements made by Quota to the Exercise of Parliamentary Activity, who pays for alimentation, transport, lodging and even culture, courses and TV signatures of federal deputies.
+  paragraph_four: On September 7 goes live the first crowdfunding campaign of the project, asking for money to pay for the team work for 2 months. In the end, the campaign raised 20% more than the original asked value, summing up more than U$25.000,00.
+  paragraph_five: 'We officially started the job in the beginning of november and it was marked, since then, by team daily meetings and for it’s fully open source content, that means, open for the community. This makes part of our guidance: more than a technology project, our goal it is to spread information and political conscience. We want to settle a new age of open data and it’s public access.'
```

Comment:
Some minor edits:

_We officially started the job in early November and since then the team holds daily meetings and has published tons of fully open source content — that is to say, open for the whole community. This makes part of our guideline: it is more than a technology project, our goal it is to spread information and political conscience. We want to settle a new age of open data and its public access._